monitor/core/listener.py

```diff
@@ -0,0 +1,70 @@
+import os
```

Comment:
same file as backend maybe use a symbolic link instead

Reply:
see also comment above about symbolic links and problems if we do not replicate the files upon physical container separation.

Reply:
this is unrelated. the symbolic link is for building the container